### PR TITLE
PartitionEditor: Show better error when can't find partition table

### PR DESCRIPTION
**Pull request by hughdavenport**

### CODE REVIEW

File: Userland/Applications/PartitionEditor/main.cpp [hunk 1]
```diff
@@ -64,7 +64,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto device_paths = get_device_paths();
 
     auto partition_model = PartitionEditor::PartitionModel::create();
-    TRY(partition_model->set_device_path(device_paths.first()));
+    auto found_it = device_paths.find_if([&](auto const& path) {
+        auto result = partition_model->set_device_path(path);
+        return !result.is_error();
+    });
+    // TODO give the user some way to create a partition table?
```
Comment (supercomputer7) [inline Userland/Applications/PartitionEditor/main.cpp:71]: We probably prefer `// FIXME:` over `// TODO`s.

Comment (hughdavenport) [reply Userland/Applications/PartitionEditor/main.cpp]: Cheers, good to be consistent :)

**Comment by hughdavenport** [comment]
Looks like the `lint_commits` test is failing due to some docker issue. I assume my change since original commit shouldn't break stuff :/

**Comment by AtkinsSJ** [comment]
> Looks like the `lint_commits` test is failing due to some docker issue. I assume my change since original commit shouldn't break stuff :/

It's not you, it's happening for every PR at the moment. People are working on fixing it.

Change looks good, let's merge this!

**Pull request merged**